public/script.js

```diff
@@ -1,216 +1,296 @@
-var CONSTANTS;
-var STATE = {
-    stop: true,
-    rate: 100,
-    current: moment().year(1990).month(0).day(1),
-    start: moment().year(1990).month(0).day(1),
-    end: moment().year(2018).month(11).day(31),
-    extCon: "extent",
-    norSouth: "n",
-    dateLoopStyle: "daily",
-    monthLoop: 0
-}
-var map;
-async function main() {
-    CONSTANTS = await readJSON("constants.json");
-    //Set default settings into the selectors and some other starting values
-    init();
-    
-    $("#animate").click(function () {//When animation button is clicked 
-        if (STATE.stop) {
-            STATE.stop = false;//Start animation
-            $("#animate").html("Stop Animation");
-        }
-        else {
-            STATE.stop = true;
-        $("#animate").html("Start Animation");
-        }
-    });
+let CONSTANTS;
+const STATE = {
+  stop: true,
+  rate: 100,
+  current: moment(),
+  start: moment(),
+  end: moment(),
+  extCon: 'extent',
+  norSouth: 'n',
+  dateLoopStyle: 'daily',
```

Comment:
I'd probably call this `temporality`. 

We don't need to do this in this PR, but I would probably also change `norSouth` to `hemi` or `hemisphere` and `extCon` to `dataType` (this has the added advantage that we could e.g., easily add an additional data type to animate).